### ISSUE TITLE
show-language plugin consistency

### ISSUE THREAD

**Issue by felixsanz** [PrismJS/prism#897]
When you use `language-html`, the plugin show-language shows "Html" and since it's an acronym, it should be **`HTML`**.

This doesn't happen on PHP code blocks. Why? Because PHP takes it's value from `"php":"PHP"` from `var Languages` (line 8 prism-show-language.js)

That variable is auto-generated using gulp task so i can't add `"html":"HTML"` there since it's not a syntax language but an alias.

Aliases are well defined in every component file, but this plugin should be noticed about them too.

Everything can be fixed by just using `data-language` but i wish this could be done without adding extra markup.

TL;DR: HTML should be capitalized according to orthography rules (and also, because PHP is)

> Probably affecting other aliases too

**Comment by felixsanz** [comment]
Maybe auto generating the variable like is done now, but also appending manually to the variable some values.

**Comment by Golmote** [comment]
Maybe add the aliases into the components file, so we can use them during the gulp generation.

**Comment by zeitgeist87** [comment]
> Maybe add the aliases into the components file, so we can use them during the gulp generation.

That would probably be best. Something like:

```js
"markup": {
	"title": "Markup",
	"aliasTitles": { "html": "HTML", "xml": "XML", "svg": "SVG", "mathml": "MathML" },
	"option": "default"
},
```

We could add them directly to the language map in the gulpfile:

```js
for (var name in data.languages[p].aliasTitles) {
	languagesMap[p] = data.languages[p].aliasTitles[name];
}
```

**Comment by Golmote** [comment]
This is exactly what I meant :) Could you take some time to implement it?

**Comment by zeitgeist87** [comment]
> This is exactly what I meant :) Could you take some time to implement it?

Sure :smile:. I've just commited it: https://github.com/PrismJS/prism/commit/54400fbaa33e005f527c37f77e401a54b8d9adcd
